content/articles/why-you-should-use-wakanda/index.md

```diff
@@ -0,0 +1,55 @@
+The Wakanda server runs and hosts the backend applications in the course of production and development. It is a multi-threaded HTTP server which incorporates server-side JavaScript APIs unmasked in REST. Everything needed is already written for the users in JavaScript. It also saves time since the user does not have to go into configuration tasks. The Wakanda server is also portable. It lowers the cost of both configuration and development by using pre-bundled components.
+
+##### JavaScript application server
+The Wakanda JavaScript app server uses the V8 engine which operates on JavaScript runtime. It exhibits a rich synchronous API to hold filesystems, sockets remote databases and workers. It support asynchronous development style.
+
+##### HTTP server
+The Wakanda HTTP server exposes the autogenerated Wakanda REST API. It is also responsible for serving static application files.
+
+##### Data abstraction layer
+The data abstraction layer is essential for data security. It helps in data manipulation through use of a JavaScript API. It exposes an automated REST API to manipulate and access data.
+
+##### Authentication (directory)
+It helps in exposing both the REST API and server side JavaScript API. It is responsible for authentication of management sessions and users.
+
+#### Wakanda studio
+The Wakanda studio helps to develop and test apps locally together with troubleshooting your environment. In this era where everyone is craving for a code editor, Wakanda has taken a different turn in coming up with an IDE. 
+
+Wakanda's main objective is to give a quick and easy way in project scaffolding. Wakanda is the solution. The philosophy behind Wakanda studio which simplifies developers lives who have decided to work with Wakanda JavaScript framework stack is called RAD (Rapid Application Development).
+
+Do you love React? You can change the scaffolding of your project to adapt to your desire. Wakanda identifies and acknowledges the three critical parts in all applications.
+
+These crucial parts are all used in the initial scaffolding. They include the backend, web and the mobile. Studio has a code editor emanating from extension updatable, an application prototype, deploy solutions, code mirror and git integration. This studio gives the user the whole view of application.
+
+##### Application scaffolding
+It composes your app by identifying templates from the configured list. This includes Backend: typescript, JavaScript. web: Angular1, Angular4 but soon ReactJS and Aurelia are coming. Mobile: Ionic1, Ionic2.
+
+##### Environmental troubleshooting
+System and configuration problems detection tools and solutions are readily prepared for you. This will help in creating value thus choosing Wakanda JavaScript framework.
+
+### Why you should use it? 
+- It is cost effective. This is due to some of its parts being reusable modules. Its easy amalgamation between the developmental phases of application development and backend prototyping.
```

Comment:
Replace the word "amalgamation" with a simpler word. The sentense `Its easy amalgamation between the developmental phases of application development and backend prototyping.` is not complete, complete it.